x-pack/plugins/apm/public/components/shared/time_comparison/index.tsx

```diff
@@ -60,11 +60,15 @@ export function TimeComparison() {
   const comparisonOptions = useMemo(() => {
     const matchingRoutes = apmRouter.getRoutesToMatch(location.pathname);
     // Only show the "Expected bounds" option in Overview and Transactions tabs
-    const showExpectedBoundsForThisTab = matchingRoutes.some(
-      (d) =>
-        d.path === '/services/{serviceName}/overview' ||
-        d.path === '/services/{serviceName}/transactions'
-    );
+    const showExpectedBoundsForThisTab =
+      !matchingRoutes.some(
+        (d) => d.path === '/services/{serviceName}/transactions/view'
+      ) &&
+      matchingRoutes.some(
+        (d) =>
+          d.path === '/services/{serviceName}/overview' ||
+          d.path === '/services/{serviceName}/transactions'
```

Comment:
Remind me: why did the previous check match `/services/{serviceName}/transactions/view`? The check only refers to `/services/{serviceName}/overview` and `/services/{serviceName}/transactions`. 

(I don't completely recall how `getRoutesToMatch` work)

Reply:
Btw. can you add a test for this in https://github.com/elastic/kibana/blob/fe190722c5352cea45947964b15670f955760fc8/x-pack/plugins/apm/public/components/shared/time_comparison/index.test.tsx ?

Reply:
This is because `matchingRoutes.map(r => r.path)` returns the following:

```
[
    "/",
    "/services/{serviceName}",
    "/services/{serviceName}/transactions",
    "/services/{serviceName}/transactions/view"
]
```

for the following link `app/apm/services/frontend/transactions/view?...`

Reply:
Added tests here [`3f37787` (#155915)](https://github.com/elastic/kibana/pull/155915/commits/3f37787478e5ea3572be4f4f32337d010ec34fc4)